src/transport/raw/tests/TestMessageHeader.cpp

```diff
@@ -23,14 +23,13 @@
  *
  */
 
+#include <gtest/gtest.h>
```

Comment:
Why these changes? It's OK to have a few include groups if the headers come from different projects.

Reply:
Good point.  I updated the groupings to comply with the order "Related header, C system headers, C++ standard library headers, other libraries' headers, your project's headers"